tests/components/tradfri/conftest.py

```diff
@@ -76,3 +76,16 @@ def mock_api_factory(mock_api):
         factory.init.return_value = factory.return_value
         factory.return_value.request = mock_api
         yield factory.return_value
+
+
+@pytest.fixture(autouse=True, scope="module")
```

Comment:
Probably don't use module scope. The module scope will make the fixture only be loaded once per module. I don't see why we'd want that here.

Reply:
Of a fixture is only used in one test file its usually more readable to keep it in that file. Shared fixtures are preferably centralized in a `conftest.py`

Reply:
I did this to make the tests in `test_coordinator.py` work but given your comment below, I'll remove that file and put back the fixture.